packages/itwinui-css/src/bridge.scss

```diff
@@ -0,0 +1,361 @@
+
+  // TODO: status-hover variables are not being used anywhere in iTwinUi v3. Investigate.
+  --iui-color-text-informational-hover: var(--iui-color-text-informational);
+  --iui-color-text-positive-hover: var(--iui-color-text-positive);
+  --iui-color-text-warning-hover: var(--iui-color-text-warning);
+  --iui-color-text-negative-hover: var(--iui-color-text-negative);
+
+  --iui-color-text: var(--kiwi-color-text-neutral-primary);
+  --iui-color-text-hover: var(--iui-color-text);
+  --iui-color-text-accent: var(--kiwi-color-text-accent-strong);
+  --iui-color-text-accent-hover: var(--iui-color-text-accent);
+  --iui-color-text-muted: var(--kiwi-color-text-neutral-secondary);
+  --iui-color-text-disabled: var(--kiwi-color-text-neutral-disabled);
+  --iui-color-text-disabled-hover: var(--iui-color-text-disabled);
+  --iui-color-text-placeholder: var(--kiwi-color-text-neutral-secondary);
+
+  &[data-iui-theme='light'] {
+    --iui-opacity-5: 0.25; // TODO: fix opacity value in iTwinUI v3
+
+    --iui-color-background-hsl: 209.21deg 17.09% 97.27%; // --kiwi-color-bg-surface-primary
+    --iui-color-foreground-hsl: 214.23deg 9.08% 15.9%; // --kiwi-color-text-neutral-primary
+    --iui-color-accent-hsl: 164.5deg 68.46% 30.05%; // --kiwi-color-icon-accent-strong
+    --iui-color-informational-hsl: 210.23deg 96.74% 36.73%; // --kiwi-color-icon-info-faded
+    --iui-color-positive-hsl: 105.38deg 97.33% 21.67%; // --kiwi-color-icon-positive-faded
+    --iui-color-warning-hsl: 39.78deg 94.44% 25.8%; // --kiwi-color-icon-attention-faded
+    --iui-color-negative-hsl: 348.19deg 79.28% 40.72%; // --kiwi-color-icon-critical-faded
+  }
+
+  &[data-iui-theme='dark'] {
+    --iui-color-background-hsl: 214.23deg 9.08% 15.9%; // --kiwi-color-bg-surface-primary
+    --iui-color-foreground-hsl: 209.2deg 40.21% 98.84%; // --kiwi-color-text-neutral-primary
+    --iui-color-accent-hsl: 166.21deg 96.14% 50.19%; // --kiwi-color-icon-accent-strong
+    --iui-color-informational-hsl: 210.52deg 76.91% 71.38%; // --kiwi-color-icon-info-faded
+    --iui-color-positive-hsl: 105.91deg 34.34% 61.19%; // --kiwi-color-icon-positive-faded
+    --iui-color-warning-hsl: 40.01deg 57.6% 59.24%; // --kiwi-color-icon-attention-faded
+    --iui-color-negative-hsl: 347.86deg 75.06% 76.69%; // --kiwi-color-icon-critical-faded
+  }
+
+  /* ---------------------------------------------------------------------------- */
+  /* Anchor */
+
+  .iui-anchor {
+    // TODO: Potentially, use underline prop.
+    text-decoration: underline;
+
+    &:where(:hover) {
+      text-decoration-color: transparent;
+    }
+
+    // Neutral color instead of accent
+    &:where(:not([data-iui-status])) {
+      color: var(--kiwi-color-text-neutral-primary);
+    }
+  }
+
+  /* ---------------------------------------------------------------------------- */
+  /* Breadcrumbs */
+
+  .iui-breadcrumbs {
+    // Neutral color instead of accent
+    --_iui-breadcrumbs-content-color-default: var(--iui-color-text);
```

Comment:
I think we can get rid of this, right? We decided to use the same set of color variables as current iTwinUI v3 implementation.